Gemfile

```diff
@@ -31,5 +31,11 @@ group :test do
   gem 'timecop'
 end
 
+group :development, :test do
+  # Pronto
+  gem 'pronto'
```

Comment:
Bundler/DuplicatedGem: Gem `pronto` requirements already given on line 24 of the Gemfile.